pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1867,6 +1867,9 @@ public CompletableFuture<TopicStatsImpl> asyncGetStats(boolean getPreciseBacklog
         stats.bytesOutCounter = bytesOutFromRemovedSubscriptions.longValue();
         stats.msgOutCounter = msgOutFromRemovedSubscriptions.longValue();
         stats.publishRateLimitedTimes = publishRateLimitedTimes;
+        stats.ongoingTxnCount = getTransactionBuffer().getOngoingTxnCount();
```

Comment:
nit: can we call getTransactionBuffer() only once ?

Reply:
it makes sense, I‘ll change it.